### DISCUSSION TITLE
Add make install

### DISCUSSION THREAD

**Discussion by wubx** [databendlabs/databend#3707] [Ideas]
Add make install for deploy 

/usr/local/databend/bin/databend-query
/usr/local/databend/bin/databend-meta
/usr/local/databend/bin/start
/usr/local/databend/bin/stop

config file in:
/usr/local/databend/etct/

Data file( if use s3 as storage  , the data dir not use )
/usr/local/databend/data/

/usr/local/databend/logs

# for startup 
/usr/local/databend/bin/start  
hit :   
start state 
log path 

# for stop 
/usr/local/databend/bin/stop

**Comment by Xuanwo** [comment]
- binary should be located at `/usr/bin/databend-xxx`.
- config should be located at `/etc/databend` (system) or `~/.config/databend` (user)
- data file should be located at `/var/lib/databend`
- log should be located at `/var/log/databend`
- we should never provide a startup to stop binary, use `systemd` instead

And I prefer all these work be done via package manager (deb, rpm, pacman...) instead of `make install`.

for reference: https://refspecs.linuxfoundation.org/FHS_3.0/fhs/index.html

**Reply by Xuanwo** [reply]
@PsiACE Maybe this issue should also be scoped under https://github.com/datafuselabs/databend/discussions/3419 ?

**Reply by PsiACE** [reply]
Yes, but for the moment we don't need to think about it.

We already have bendctl to handle basic installation and management.
